de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/JsonHandler.java

```diff
@@ -52,11 +57,16 @@ protected JsonHandler(ScriptHandler scriptHandler, ExecutionContext exec) {
     }
   }
 
-  // TODO: conversion command? maybe create a new Class instead of using a map
+  // Method loads parameter data from previous models into workspace of current model
+  // according to the list of Joiner-Relations
   public void applyJoinRelation(FskPortObject fskObj, List<JoinRelationAdvanced> joinRelationList,
       String suffix) throws Exception {
 
+    // figure out which parameters from JoinRelations belong to which model to avoid loading
+    // a json file twice
     if (joinRelationList != null) {
+      // get all relevant parameters -> assign to Map with key=jsonPath
+      Map<String, List<JoinerObject>> sourceTargetPathMap = new HashMap();
       for (JoinRelationAdvanced joinRelation : joinRelationList) {
         String targetParameter = joinRelation.getTargetParam();

```

Comment:
Personal preference. If getters do not have any logic I would use them straight instead of assigning them to local variables to keep the stack clean.